### ISSUE TITLE
How to get started with navigation in Ignite2?

### ISSUE THREAD

**Issue by pekkanikolaus** [infinitered/ignite#952]
Still a newbie to RN and ignite, and still struggling to understand the various navigation options available (+ then coupled with Redux! Arrrrgh!)

In Ignite1, we had a ready-made navigation (with the dev screens) to build on. Now in I2, that part of the system seems to have moved into the separate "dev screens" container, which is great - but now I'm stumped as to how to get started with setting up my own nav. 

Is there an Ignite way to do this, or do I simply integrate whichever nav solution I decide upon? Do I have to be aware of any conflicts between whatever nav solution I choose, and what I2 is using for the dev screens?

I can't find anything on this in the documentation, and there don't seem to be any navigation-related boilerplates ready yet.

**Comment by GantMan** [comment]
Heya!  In short, I'll trade you a start up screen share for an hour, if you'll help document/blog what you learn.   I'm trying to get the docs friendly to help people like you get started, and could use help with your perspective 👍

**Comment by pekkanikolaus** [comment]
That sounds great! Will contact you right away.

**Comment by pekkanikolaus** [comment]
@GantMan that sounds great, happy to do that! I sent you an E-Mail through your contact form. If it doesn't make it, hit me up directly through my E-Mail address in my Github profile. Looking forward to it!

**Comment by GantMan** [comment]
email sent!

**Comment by GantMan** [comment]
closing this bc we had an awesome chat!  I hope to see links to anything you publish.  Cheers!

**Comment by jmuziki** [comment]
@pekkanikolaus I happen to be in the same noob situation. Could you advise on the resources you found helpful with regard to navigation in I2? Alternatively, were you able yo document/blog your session with @GantMan

**Comment by pekkanikolaus** [comment]
@jmuziki I'm intending to do a detailed writeup on how I got it working, but it'll be a while!

Short version though. There's a very instructive video on how to get started with I2 here: https://youtu.be/lv00UOXxy0g

If you create an ignite project using the `ir-next` boilerplate, you'll get an empty project with React Navigation already set up:

> ignite new MyApp -b ir-next

That'll give you a `Navigation/AppNavigation.js` ready to tinker with. Now in my case I wanted to set up a tab navigation. To do that, I removed the `StackNavigator` defined there and replaced it by a `TabNavigator`. I've since learned [that is not the ideal way to go](https://github.com/infinitered/ignite/issues/964) - the root navigation is always supposed to be a `StackNavigator` and the `TabNavigator` is supposed to be a child of the root nav. Robin Harris, the guy who pointed that out to me in the github issue I linked to, has [an example of an I2 install with a Redux-based tab nav.](https://github.com/HarrisRobin/ignite-react-navigation-redux-example) That's what I'm planning to look at next, and adjust my code accordingly.

Hope that helps you get started!
